[Deployment] Node upload yaml fix

pkommoju opened the pull request:
This PR has three changes:

    Corrections to NCM yaml file (k8s)
    Delete command in Services (k8s)
    A fix which disappeared (either as a mixup or somehow missed merging)

Abandon previous PR(#616) and branch (node_upload_yaml_fix).
It has already been reviewed in the abandoned PR and branch.

xieus commented on docs/modules/ROOT/pages/db_services/alcor-transactions.adoc:
```diff
@@ -0,0 +1,161 @@
+= Transactions in Alcor
+Prasad Kommoju <pkommoju@futurewei.com>
+v.01, 2021-05-05
+:toc: right
+
```
The file is still alive :-)

pkommoju commented:
Can't understand this, on my local machine, git status -uno showed this:
pkommoju@pkommoju-WA01 MINGW64 /d/alcor (node_upload_yaml_fix_02)
$ gso
On branch node_upload_yaml_fix_02
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   kubernetes/services/deleteYaml.sh
        modified:   kubernetes/services/network_config_manager.yaml
        modified:   services/node_manager/src/main/java/com/futurewei/alcor/nodemanager/service/implement/NodeFileLoader.java

How to get rid of it then? Is it possible/easier to do a cherrypick merge, leaving this file out?

cj-chung reviewed (approved):
LGTM
